From 824bffcc3c7fb431f97833eb38a5d9bad0631313 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:15 -0400
Subject: [usb] xhci: remove unused argument from xhci_giveback_urb_in_irq()

Message-id: <1410887463-20674-64-git-send-email-dzickus@redhat.com>
Patchwork-id: 95204
O-Subject: [RHEL7 PATCH 063/591] xhci: remove unused argument from xhci_giveback_urb_in_irq()
Bugzilla: 1032371
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1032371

commit 07a37e9e425a3176875a0c91e59ae8bc65d2ddb9
Author: Xenia Ragiadakou <burzalodowa@gmail.com>
Date:   Mon Sep 9 13:29:45 2013 +0300

    xhci: remove unused argument from xhci_giveback_urb_in_irq()

    This patch removes the "adjective" argument from xhci_giveback_urb_in_irq(),
    since it is not used in the function anymore.

    Signed-off-by: Xenia Ragiadakou <burzalodowa@gmail.com>
    Acked-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/xhci-ring.c b/drivers/usb/host/xhci-ring.c
index e187943..0a35bbd 100644
--- a/drivers/usb/host/xhci-ring.c
+++ b/drivers/usb/host/xhci-ring.c
@@ -726,7 +726,7 @@ static void xhci_stop_watchdog_timer_in_irq(struct xhci_hcd *xhci,
 
 /* Must be called with xhci->lock held in interrupt context */
 static void xhci_giveback_urb_in_irq(struct xhci_hcd *xhci,
-  struct xhci_td *cur_td, int status, char *adjective)
+  struct xhci_td *cur_td, int status)
 {
  struct usb_hcd *hcd;
  struct urb *urb;
@@ -891,7 +891,7 @@ remove_finished_td:
   /* Doesn't matter what we pass for status, since the core will
    * just overwrite it (because the URB has been unlinked).
    */
-  xhci_giveback_urb_in_irq(xhci, cur_td, 0, "cancelled");
+  xhci_giveback_urb_in_irq(xhci, cur_td, 0);
 
   /* Stop processing the cancelled list if the watchdog timer is
    * running.
@@ -1001,7 +1001,7 @@ void xhci_stop_endpoint_command_watchdog(unsigned long arg)
     if (!list_empty(&cur_td->cancelled_td_list))
      list_del_init(&cur_td->cancelled_td_list);
     xhci_giveback_urb_in_irq(xhci, cur_td,
-      -ESHUTDOWN, "killed");
+      -ESHUTDOWN);
    }
    while (!list_empty(&temp_ep->cancelled_td_list)) {
     cur_td = list_first_entry(
@@ -1010,7 +1010,7 @@ void xhci_stop_endpoint_command_watchdog(unsigned long arg)
       cancelled_td_list);
     list_del_init(&cur_td->cancelled_td_list);
     xhci_giveback_urb_in_irq(xhci, cur_td,
-      -ESHUTDOWN, "killed");
+      -ESHUTDOWN);
    }
   }
  }
-- 
1.7.1